src/h5web/visualizations/heatmap/interpolators.ts

```diff
@@ -102,10 +103,10 @@ export const INTERPOLATORS = {
   ...DIVERGING,
 };
 
-export const INTERPOLATOR_GROUPS = {
-  Common: MOST_USED,
-  'Single hue': SINGLE_HUE,
-  'Multi hue': MULTI_HUE,
-  Cyclical: CYCLICAL,
-  Diverging: DIVERGING,
+export const COLORMAP_GROUPS = {
+  Common: Object.keys(MOST_USED) as ColorMap[],
+  'Single hue': Object.keys(SINGLE_HUE) as ColorMap[],
+  'Multi hue': Object.keys(MULTI_HUE) as ColorMap[],
+  Cyclical: Object.keys(CYCLICAL) as ColorMap[],
+  Diverging: Object.keys(DIVERGING) as ColorMap[],
```

Comment:
Hmm you must be getting a cyclic dependency error from using `ColorMap` here, aren't you??

Reply:
You should be able to get around it by casting `option` to `ColorMap` in `ColorMapOption`.